src/js/index.js

```diff
@@ -14,11 +14,11 @@ import { ResultRoute } from './routes/result';
 import { Search } from './components/Search.jsx';
 
 // load styles into bundle
-import '../css/custom.css';
 import '../fonts/fonts.css';
 import 'normalize.css/normalize.css';
 import '@blueprintjs/core/dist/blueprint.css';
 import '@blueprintjs/table/dist/table.css';
+import '../css/custom.css';
```

Comment:
can't believe I didn't load ours last - maybe lots of "!importants" to remove at a later date...